kernel/events.c

```diff
@@ -336,3 +344,24 @@ uint32_t z_vrfy_k_event_wait_all(struct k_event *event, uint32_t events,
 }
 #include <syscalls/k_event_wait_all_mrsh.c>
 #endif
+
+#ifdef CONFIG_OBJ_CORE_EVENT
+static int init_event_module(void)
```

Comment:
Style: the term "module" is pretty heavily overloaded, and this doesn't really represent a "module" abstraction in the way the word gets used anywhere else.  It's a 100% local name, would be clearer as "init_obj_core_list()" or whatever as that's all it's actually doing.

Reply:
Can do.